guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb

```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Code in section 2.1 does not contain any specific instructions to use unique values for the railway lines, but the railway lines are in different color. How does this work?</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>